### PR TITLE
[Enhancement] Change misleading participation component label

### PR DESCRIPTION
**Pull request by kloessst**
### Checklist
- [x] Client: I translated all the newly inserted strings into German and English

### Motivation and Context

The label was misleading according to @krusche 

### Steps for Testing

1. Review the new labels

### CODE REVIEW

File: src/main/webapp/i18n/de/exercise.json [hunk 1]
```diff
@@ -102,7 +102,7 @@
             "score": "Punktzahl",
             "duration": "Dauer",
             "showBuildFailed": "Nur fehlgeschlagene Builds anzeigen",
-            "showFailed": "Nur fehlgeschlagene Einreichungen (ohne Ergebnis) anzeigen",
+            "showFailed": "Nur letzte Einreichungen ohne Ergebnis anzeigen",
```
Comment (fde312) [inline src/main/webapp/i18n/de/exercise.json:105]:
```suggestion
            "showFailed": "Nur die letzte Einreichungen ohne Ergebnis anzeigen",
```

File: src/main/webapp/i18n/en/exercise.json [hunk 2]
```diff
@@ -102,7 +102,7 @@
             "score": "Score",
             "duration": "Duration",
             "showBuildFailed": "Show Build Failed",
-            "showFailed": "Show Failed Submissions (without Result)",
+            "showFailed": "Show Latest Submission without Result",
```
Comment (fde312) [inline src/main/webapp/i18n/en/exercise.json:105]: why uppercase?
```suggestion
            "showFailed": "Show latest Submission without Result",
```

**Review by fde312 — CHANGES_REQUESTED** [review]
Some suggestions

File: src/main/webapp/i18n/de/exercise.json (hunk 1 above)
Comment (kloessst) [reply src/main/webapp/i18n/de/exercise.json:105]: _Nur die letzte Einreichungen.._ ist kein deutscher Satz 😄 . Ich denke das passt schon so.

File: src/main/webapp/i18n/en/exercise.json (hunk 2 above)
Comment (kloessst) [reply src/main/webapp/i18n/en/exercise.json:105]: Other labels on this site also use capital letters. Followed the convention.

**Review by fde312 — APPROVED** [review]
Tested it locally and works for me. The label has its new name. Good job 🚀 
image
image

**Review by krusche — APPROVED** [review]

**Pull request merged**